torch/jit/frontend.py

```diff
@@ -18,9 +19,15 @@
 )
 from torch._utils_internal import get_source_lines_and_file
 from torch.jit._monkeytype_config import monkeytype_trace, get_qualified_name
-from torch._jit_internal import SourceContext, should_drop, is_static_fn
+from torch._jit_internal import SourceContext, should_drop, is_static_fn, FunctionModifiers  # noqa: F401
 import torch.jit.annotations
 
+_IS_ASTUNPARSE_INSTALLED = True
+try:
+    import astunparse  # type: ignore
+except ImportError:
+    _IS_ASTUNPARSE_INSTALLED = False
```

Comment:
Can we make the logic a bit more defensive? ie, default value of `_IS_ASTUNPARSE_INSTALLED` is `False` and only set to true when import succeeds.